app/src/main/res/values/strings.xml

```diff
@@ -304,55 +304,55 @@
-    <string name="location_service_disabled_notice">Location Service is disabled</string>
-    <string name="wifi_disabled_notice">Wi-Fi is disabled</string>
+    <string name="location_service_disabled_notice">Location service off</string>
+    <string name="wifi_disabled_notice">Wi-Fi off</string>
     <string name="write_system_settings_permission_required_notice">Needs permission to modify system settings</string>
-    <string name="connecting_to_client">Connecting to Client</string>
+    <string name="connecting_to_client">Connecting to Client…</string>
     <string name="client_offline_notice">The remote is unavailable</string>
     <string name="matching_client_question">Is this the client you are looking for?</string>
     <string name="verify_client">Verify Client</string>
     <string name="client_details">Client Details</string>
     <string name="client_model">Model</string>
     <string name="percent_sign" translatable="false">%</string>
-    <string name="waiting_receiver_approval_notice">Waiting for the approval of receiver</string>
+    <string name="waiting_receiver_approval_notice">Awaiting recipient approval</string>
     <string name="incoming">Incoming</string>
     <string name="outgoing">Outgoing</string>
-    <string name="error_communication_unknown">An unknown communication error occurred</string>
-    <string name="error_client_no_address">The client has no address (try manually connecting)</string>
-    <string name="error_protocol_unknown">An unknown protocol error occurred</string>
+    <string name="error_communication_unknown">Unknown communication error</string>
+    <string name="error_client_no_address">The client has no address (try connecting manually)</string>
+    <string name="error_protocol_unknown">Unknown protocol error</string>
     <string name="completed">Completed</string>
-    <string name="run_in_background">Run in Background</string>
+    <string name="run_in_background">Run in Background…</string>
     <string name="selections">Selections</string>
-    <string name="preparing">Preparing</string>
-    <string name="web_share_description">An interface for file-sharing via browsers</string>
+    <string name="preparing">Preparing…</string>
+    <string name="web_share_description">An interface for file-sharing via web browsers</string>
     <string name="share_on_web">Share on Web</string>
     <string name="sharing_on_web">Sharing on Web</string>
     <string name="songs">Songs</string>
     <string name="videos">Videos</string>
     <string name="images">Images</string>
     <string name="folders">Folders</string>
-    <string name="web_share_empty_notice">Nothing is being shared yet. Share something on the app and come back here!</string>
-    <string name="send_using_web_share_success_notice">Successfully sent \'%s\'</string>
+    <string name="web_share_empty_notice">Share something from the app first.</string>
+    <string name="send_using_web_share_success_notice">Sent \'%s\'</string>
```

Comment:
What is the policy on `'` vs `"`?

Reply:
I think we can go with the former for now. I will probably turn them into HTML tags in the feature. 

(Just haven't looked into how it behaves on older versions of Android.)

Reply:
```suggestion
    <string name="send_using_web_share_success_notice">Sent \"%s\"</string>
```